oximeter/oximeter/src/bin/oximeter-schema.rs

```diff
@@ -9,7 +9,7 @@
 use anyhow::Context as _;
```

Comment:
Not a huge deal, but I might consider moving this binary to the new `oximeter-schema` crate. It's really useful for developing a new schema, but not many other situations!

Reply:
Oh yeah, that sounds good!

Reply:
Done.